terraform/aws/hashistack.pkr.hcl

```diff
@@ -0,0 +1,70 @@
+variable "hcp_bucket_name"{
+  default="hashistack"
+}
+
+variable "image_name"{
+  default="hashistack"
+}
+
+data "amazon-ami" "aws_base" {
+  filters = {
+    architecture                       = "x86_64"
+    "block-device-mapping.volume-type" = "gp2"
+    name                               = "ubuntu/images/hvm-ssd/ubuntu-xenial-16.04-amd64-server-*"
+    root-device-type                   = "ebs"
+    virtualization-type                = "hvm"
+  }
+  most_recent = true
+  owners      = ["099720109477"]
```

Comment:
As long as we're here, can we leave a comment on this account ID saying who's it is?

Reply:
Yeah. That's the default AWS account.  I'll add a comment on it.